src/EditorFeatures/DiagnosticsTestUtilities/Diagnostics/AbstractDiagnosticProviderBasedUserDiagnosticTest.cs

```diff
@@ -10,21 +10,22 @@
 using Microsoft.CodeAnalysis.CodeActions;
 using Microsoft.CodeAnalysis.CodeFixes;
 using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Editor.UnitTests.CodeActions;
 using Microsoft.CodeAnalysis.Editor.UnitTests.Workspaces;
 using Microsoft.CodeAnalysis.Text;
 using Microsoft.CodeAnalysis.UnitTests.Diagnostics;
 using Roslyn.Test.Utilities;
 using Roslyn.Utilities;
 using Xunit;
 
+namespace Microsoft.CodeAnalysis.Editor.UnitTests.Diagnostics
+{
 #if CODE_STYLE
-using Microsoft.CodeAnalysis.Internal.Options;
+    using TestParametersOptions = IOptionsCollection;
 #else
-using Microsoft.CodeAnalysis.Options;
+    using TestParametersOptions = IDictionary<CodeAnalysis.Options.OptionKey2, object>;
```

Comment:
Both sides don't have an IOptionsCollection now? I've lost track...

Reply:
Yes, they do but we have tons of IDE tests (not ported ones) which are still passing in `IDictionary<CodeAnalysis.Options.OptionKey2, object>` to the test helpers. It would be an easy cleanup to switch them over in a follow-up PR. I have added a tracking item at the end of https://github.com/dotnet/roslyn/issues/41462